src/pages/studyView/styles.scss

```diff
@@ -18,6 +18,38 @@
     .btn:active:focus {
         box-shadow: none; 
     }
+
+    .Select {
```

Comment:
Doesn't the style come with the component already?

Reply:
seems not

Reply:
The style needs to be updated, especially the line hight. Please see the style in coExpression folder under resultsView as example.